DataConnectors/microsoft-logstash-output-azure-loganalytics/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure Log Analytics output plugin for Logstash 
+
+Azure Sentinel provides a new output plugin for Logstash. Using this output plugin, you will be able to send any log you want using Logstash to the Azure Sentinel/Log Analytics workspace
+Today you will be able to send messages to custom logs table that you will define in the output plugin. 
+Getting started with Logstash 
+
+Azure Sentinel output plugin uses the rest API integration to Log Analytics, in order to ingest the logs into custom logs tables [What are custom logs tables] 
+
+Plugin version: v1.0.0 
+Released on: 2020-04-30 
+
+## Installation
+
+Azure Sentinel provides Logstash output plugin to Log analytics workspace. 
+Install the microsoft-logstash-output-azure-loganalytics, use Logstash Working with plugins document. 
```

Comment:
Link to Working with Plugins: https://www.elastic.co/guide/en/logstash/current/working-with-plugins.html